.github/workflows/ci-weekly.yml

```diff
@@ -48,3 +48,25 @@ jobs:
 
     - name: Execute unit tests
       run: lerna run test --stream
+
+  code_scan:
+    name: Validate Security
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 2
+        
+    - run: git checkout HEAD^2
```

Comment:
why HEAD^2?

Reply:
Refer to [this](https://github.com/microsoft/fast-dna/pull/3234#discussion_r436038835) for the answer